_data/pages/navigation.yml

```
@@ -46,6 +46,8 @@ sidenav:
         href: /pages/documentation/why-use-pages/
       - text: Access and permissions
         href: /pages/documentation/access-permissions/
+      - text: Adding a user to an Organization
+        href: /pages/documentation/add-user
```

Match the href with the permalink in the file

```suggestion
        href: /pages/documentation/adding-users/
```